package.json

```diff
@@ -396,6 +396,16 @@
           "orders-storage.custom-fields.item.delete"
         ]
       },
+      {
```

Comment:
hm, what is it?

Reply:
https://github.com/folio-org/ui-orders/pull/1597#discussion_r1582599172

Reply:
Removed the permission according to your comment `"templates.item.post"`

Reply:
you removed subpermissions and created Settings (Orders): Can create, edit, view and delete custom fields ?